test-browser/tests/ballot.js

```diff
@@ -13,19 +13,7 @@ function runTests (browser, testData) {
     .url('http://127.0.0.1:8080/#version=builtin')
     .waitForElementVisible('.newFile', 10000);
   browser.assert.notEqual(testData, null);
-  // TODO add Ballot tests. -> setValue('#input textarea', ... ) is not working properly with that contract.
-  /* testBallot(browser, testData.ballot.sources.Untitled, function () {
+  contractHelper.testContracts(browser, testData.ballot.sources.Untitled.replace(/(\n)/g, ' '), ['Ballot'], function () { // removing \n to get rid of autoclose bracket
```

Comment:
yes, this is needed to get rid of the auto close bracket feature of ace editor.
We could turn off this feature for tests but this will need more changes